src/modules/fetchlike.js

```diff
@@ -0,0 +1,7 @@
+export default async function fetchlike(key) {
+  return fetch(
+    `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${key}/likes`,
+  )
+    .then((res) => res.json())
+    .then((output) => output);
+}
```

Comment:
- [ ] Same here, let's use `arrow` syntax instead of JavaScript Function Expressions.

- [ ] you should Use `async` and `await` with the `fetch` method to fetch like but I can see that you used Promises. Kindly avoid this

- This is an example:



```
  const getScores = async () => {
     let response = await fetch(requestedURL);
        let data = await response.json();
        renderScores(data.result);
  } 
```

- For more documentation, you can check  [Promises vs Async Await](https://www.loginradius.com/blog/engineering/callback-vs-promises-vs-async-await/)

Reply:
done for the arrow function, but for the promise it is part of the project requirement as mentioned here:
<img width="448" alt="callbacks" src="https://user-images.githubusercontent.com/77294268/212458202-fe964a6f-f894-4209-af8c-6e200b0fe23c.png">